crates/kitsune_p2p/kitsune_p2p/src/spawn/actor/bootstrap.rs

```diff
@@ -0,0 +1,41 @@
+lazy_static::lazy_static! {
+    static ref CLIENT: reqwest::Client = reqwest::Client::new();
+}
```

Comment:
```suggestion
use once_cell::sync::Lazy;

static CLIENT: Lazy<reqwest::Client> = Lazy::new(|| reqwest::Client::new());
```

once_cell is the new lazy_static - it's more efficient on the back-end.